### PR TITLE
Codecyfix

### PR DESCRIPTION
**Pull request by studiotbt**
minor css size issue, 0 to 0px.

### CODE REVIEW

**Comment by studiotbt** [comment]
not clear what is wrong with this

File: button/popup.css [hunk 1]
```diff
@@ -292,11 +296,11 @@ http://matthewlein.com/ceaser/
 }
 
 .sitetitle {
-    margin-top: 3px;
     display: block;
-    left: -10px;
+    height: 35px;
     position: inherit;
-    padding: 10px 0 0 15px;
+    margin: 15px 10px 0px 5px;
```
Comment (tomasko126) [inline button/popup.css]: Codacy Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/tomastaro/notemark/file/2651364002/issues/source?bid=2884821&fileBranchId=3324709#l302)

File: button/popup.css [hunk 2]
```diff
@@ -282,8 +286,8 @@ http://matthewlein.com/ceaser/
 
 .removenote2 {
    transition: 0.3s cubic-bezier(0.075, 0.820, 0.165, 1.000); /* easeOutCirc */;
-   height: 0;
-   padding: 0;
+   height: 0px;
```
Comment (tomasko126) [inline button/popup.css]: Codacy Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/tomastaro/notemark/file/2651364002/issues/source?bid=2884821&fileBranchId=3324709#l289)

File: button/popup.css [hunk 3]
```diff
@@ -56,25 +56,29 @@ http://matthewlein.com/ceaser/
...
     border-top: 1px solid #e3e3e3;
     border-bottom: 1px solid #e3e3e3;
     position: relative;
     z-index: 1;
     width: 100%;
     cursor: pointer;
+}
 
+.site:last-child {
+    border-bottom: 0px;
```
Comment (tomasko126) [inline button/popup.css]: Codacy Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/tomastaro/notemark/file/2651364002/issues/source?bid=2884821&fileBranchId=3324709#l80)

File: button/popup.css [hunk 4]
```diff
@@ -56,25 +56,29 @@ http://matthewlein.com/ceaser/
...
     background: #fff;
+    float: left;
     cursor: default;
     border-top: 1px solid #e3e3e3;
 }
 
 .site { /* each note area */
     height: 50px; /* prev hieght 40px */
-    padding: 2px 0 2px 0;
+    margin: 2px 0px 0px 0px;
```
Comment (tomasko126) [inline button/popup.css]: Codacy Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/tomastaro/notemark/file/2651364002/issues/source?bid=2884821&fileBranchId=3324709#l70)

File: button/popup.css [hunk 5]
```diff
@@ -282,8 +286,8 @@ http://matthewlein.com/ceaser/
 
 .removenote2 {
    transition: 0.3s cubic-bezier(0.075, 0.820, 0.165, 1.000); /* easeOutCirc */;
-   height: 0;
-   padding: 0;
+   height: 0px;
+   padding: 0px;
```
Comment (tomasko126) [inline button/popup.css]: Codacy Issue found: [Values of 0 shouldn't have units specified.](https://www.codacy.com/app/tomastaro/notemark/file/2651364002/issues/source?bid=2884821&fileBranchId=3324709#l290)

**Comment by tomasko126** [comment]
@studiotbt I will close this one in favor of #71.

**Pull request closed without merging**